src/mito2/src/manifest/manager.rs

```diff
@@ -373,20 +306,54 @@ impl RegionManifestManagerInner {
         Ok(version)
     }
 
+    /// Retrieves the current [RegionManifest].
+    pub fn manifest(&self) -> Arc<RegionManifest> {
+        self.manifest.clone()
+    }
+
     /// Returns total manifest size.
-    pub(crate) fn total_manifest_size(&self) -> u64 {
+    pub fn manifest_usage(&self) -> u64 {
         self.store.total_manifest_size()
     }
-}
 
-impl RegionManifestManagerInner {
+    /// Returns true if a newer version manifest file is found.
```

Comment:
Considering concurrency, this function looks like could not provide a strong guarantee of the result, should we document it?